### ISSUE TITLE
0.6.0 - Cannot read property 0 of null.

### ISSUE THREAD

**Issue by MichalKoscielniak** [Microsoft/vsts-zendesk-app#41]
I have just installed the app ugraded to v2 ( was waiting for this for quite a while ), and unfortunately it does not work. When I click "New work item", it's creating an empty modal with no fields to populate. In the console i can see the error "Cannot read property 0 of null ( occuring in modal.js )". Similiar thing happens when i open "Link to work item" modal - it presents the same error as with new work item. Tested od both Chrome and Edge browsers.
![zendesk_issue_1](https://user-images.githubusercontent.com/34438784/33824280-292f7660-de5e-11e7-956a-07952755c5da.png)
![zendesk_issue_2](https://user-images.githubusercontent.com/34438784/33824281-29496660-de5e-11e7-8c7f-7eea1273eaf5.png)
![zendesk_issue_3](https://user-images.githubusercontent.com/34438784/33824282-296151bc-de5e-11e7-9cf2-7cb812a01653.png)

**Comment by MichalKoscielniak** [comment]
Found the answer on different thread. The issue is gone when i set additional credentials on PAT, as per below suggestion.
![zendesk_issue_resolved](https://user-images.githubusercontent.com/34438784/33826825-792c0298-de66-11e7-9f89-8007ddbc9cf5.png)

**Comment by webtechy** [comment]
Agreed this was a bug as was seeing a similar issue with resolution above.

**Comment by dbr2hb** [comment]
Hi.  I have tried granting the rights above and still am getting the error.  Any suggestions?

**Comment by BenAffleckIsBatman** [comment]
@dbr2hb @willsmythe Got the same issue. I solved it by giving the following scope rights on the PAT in VSTS:

- Team dashboards (read)
- Work items (read and write)
- Work item search (read)

I think the documentation is wrong.

**Comment by pmosupport** [comment]
hi

I cannot seem to get the integration working..

Comments updated at either end do not flow through...

has anyone else had the issue bfore??

**Comment by ekozelek** [comment]
Leaving a note for anyone in the future who encounters this error - I was receiving it on the latest version using a PAT with work item read & write scopes. The fix for me was to give my PAT full access.
